src/Bumblebee.IntegrationTests/Extensions/VerificationTests.cs

```diff
@@ -1,15 +1,40 @@
 ﻿using System;
+using System.IO;
+using System.Reflection;
+
 using Bumblebee.Exceptions;
 using Bumblebee.Extensions;
 using Bumblebee.Interfaces;
+using Bumblebee.Setup;
+
 using FluentAssertions;
+
 using NSubstitute;
 
 using NUnit.Framework;
 
+using OpenQA.Selenium;
+
+// ReSharper disable InconsistentNaming
+
 namespace Bumblebee.IntegrationTests.Extensions
 {
-	// ReSharper disable InconsistentNaming
+	public class TestClickable : IClickable
```

Comment:
In VerificationTests.cs, I will move TestClickable down below the Test so that it reads more like a novel as described in Clean Code.